server/src/lib.rs

```diff
@@ -70,7 +66,9 @@ impl From<EngineError> for u32 {
         }
     }
 }
-
+lazy_static::lazy_static! {
+    static ref CLIENT: Arc<rpc::client::ClientAsync> = Arc::new(rpc::client::ClientAsync::new());
```

Comment:
Maybe you can use Box::new()? I remember it is on the heap.

Reply:
It should be possible to remove `Arc` directly here, don't care if it's on the heap

Reply:
@uran0sH Any idea?

Reply:
> It should be possible to remove `Arc` directly here, don't care if it's on the heap

maybe it's ok

Reply:
@Tricky511 Now you can remove the lazy_static.